recipes/openscenegraph/all/conanfile.py

```diff
@@ -186,6 +194,8 @@ def _configured_cmake(self):
 
         cmake.definitions["OSG_TEXT_USE_FONTCONFIG"] = self.options.use_fontconfig
 
+        cmake.definitions["OPENGL_PROFILE"] = str(self.options.get_safe("opengl_profile", "gl2")).upper()
```

Comment:
This option is not deleted, lets try to keep it simple 

```suggestion
        cmake.definitions["OPENGL_PROFILE"] = str(self.options.opengl_profile).upper()
```